README.md

```diff
@@ -1,7 +1,73 @@
+| [vaadin-upload](https://github.com/vaadin/vaadin-upload) | ![Bower version](https://badge.fury.io/bo/vaadin-upload.svg) | [![Build Status](https://travis-ci.org/vaadin/vaadin-upload.svg?branch=master)](https://travis-ci.org/vaadin/vaadin-upload) | ![Polymer 2 supported](https://img.shields.io/badge/Polymer2-supported-blue.svg) | [![npm version](https://badge.fury.io/js/%40vaadin%2Fvaadin-upload.svg)](https://badge.fury.io/js/%40vaadin%2Fvaadin-upload) |
+
+Components marked with a `*` are part of the [Vaadin Pro Subscription](https://vaadin.com/pricing), other components are licensed under the Apache License 2.0. 
+
+
+## Learn more
+
+See the list of components, demos and documentation, instructions how to get started, video tutorials, application demos and more by visiting:
+
+### [vaadin.com/components ›](https://vaadin.com/components)
+
+---
+
+## Contributing
+
+We would absolutely love to get the community involved, and we welcome any form of contributions – comments and questions on different communication channels, issues and pull request in the repositories, and anything that you build and share using our components.
+
+### Get in touch with the team
+
+The chat and the forum category are dedicated channels for this project, while the Twitter, Facebook and G+ pages are common for all Vaadin related discussion (including the Java framework).
+
+- [Chat](https://gitter.im/vaadin/web-components)
+- [Forum](https://vaadin.com/forum/#!/category/9848927)
+- [Twitter](https://twitter.com/vaadin)
+- [Facebook](https://www.facebook.com/vaadin/)
+- [G+](https://plus.google.com/communities/108116678608923665301)
+
+### Some ways to help:
+
+- **Test the components and provide feedback**: We would love to hear your feedback on anything related to the components, like features, API and design. The best way to start is by [trying them out](https://vaadin.com/components/browse). And to get a quick response, either drop a question/comment on the chat or open an issue in GitHub.
+- **Report bugs**: File issues for the components in their respective GitHub projects. You can find links to them from the list in this readme.
+- **Send pull requests**: If you want to contribute code, check out the development instructions below.
+
+We encourage you to read the [contribution instructions by GitHub](https://guides.github.com/activities/contributing-to-open-source/#contributing) also.
+
+## Development
+
+If you want to develop the components yourself see the [development instructions](DEVELOPMENT.md) that apply to all Vaadin components.
```

Comment:
Please include the `DEVELOPMENT.md` to this PR

Reply:
Ping @Artur-

Reply:
Done